.github/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -26,17 +27,26 @@ body:
     validations:
       required: true
   - type: textarea
-    id: description
+    id: issue
     attributes:
-      label: Describe the Issue and Proposed Fix
-      description: Describe the issue and what you think needs to be added, updated, or otherwise changed. Be as descriptive as possible.
-      placeholder:  
+      label: The Issue or Context
```

```suggestion
      label: The issue or context
```
I'd prefer not to capitalize the fields themselves